<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reminder_fee</class>
 <widget class="QWidget" name="reminder_fee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>reminder_fee</string>
  </property>
  <widget class="QDateEdit" name="today_dateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>438</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>440</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前日期</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>741</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="query_not_paid_btn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询未缴费名单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reminder_pay_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>440</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>用email提醒业主缴费</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fee_species_comboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>物业费</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>停车费</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
